geopyspark/geotrellis/catalog.py

```diff
@@ -4,19 +4,19 @@
 import json
 from collections import namedtuple
 from urllib.parse import urlparse
-
+from itertools import groupby
 from shapely.geometry import Polygon, MultiPolygon, Point
 from shapely.wkt import dumps
 import shapely.wkb
 
-from geopyspark import map_key_input, get_spark_context
+from geopyspark import map_key_input, get_spark_context, scala_companion
 from geopyspark.geotrellis.constants import LayerType, IndexingMethod, TimeUnit
 from geopyspark.geotrellis.protobufcodecs import multibandtile_decoder
 from geopyspark.geotrellis import Metadata, Extent, deprecated, Log
 from geopyspark.geotrellis.layer import TiledRasterLayer
 
 
-__all__ = ["read_layer_metadata", "get_layer_ids", "read_value", "query", "write"]
+__all__ = ["read_layer_metadata", "get_layer_ids", "read_value", "query", "write", "AttributeStore"]
```

Comment:
Is `AttributeStore` something the users will be working with in the future? Because I'm not sure if it makes sense to add it to `__all__` if that's not the case.